lib/matplotlib/rcsetup.py

```diff
@@ -988,7 +988,12 @@ def validate_webagg_address(s):
     raise ValueError("'webagg.address' is not a valid IP address")
 
 
-validate_axes_titlelocation = ValidateInStrings('axes.titlelocation', ['left', 'center', 'right'])
+validate_axes_titlelocation = ValidateInStrings('axes.titlelocation',
```

Comment:
this one is not needed?

Reply:
What do you mean?

Reply:
I don't think this function is ever used?

Reply:
Should be for the axes title location at line 1180